Doppler.Sap.Test/CreateOrUpdateBusinessPartnerHandlerTest.cs

```diff
@@ -20,7 +20,7 @@ public class CreateOrUpdateBusinessPartnerHandlerTest
     {
         [Fact]
 
-        public void CreateOrUpdateBusinessPartnerHandler_ShouldBeThownsAnExcpetion_WhenCountryCodeNotAROrUS()
+        public async void CreateOrUpdateBusinessPartnerHandler_ShouldBeThownsAnExcpetion_WhenCountryCodeNotAROrUS()
```

Comment:
I think that if this method is `async` should return a `Task`, 
https://docs.microsoft.com/en-us/archive/msdn-magazine/2013/march/async-await-best-practices-in-asynchronous-programming#avoid-async-void

Reply:
You are right I will change it

Reply:
Done